src/learning.py

```diff
@@ -292,7 +292,7 @@ def compute_cost_matrix(
     return cost_df
 
 
-def get_normalizing_constants(hall_df, student_df, art_df):
+def get_normalizing_constants(hall_df, student_df, art_df, curator_df):
```

Comment:
For this function (and any other functions you added) you'll need to fix/create a docstring.  These are useful for our future selves when we've long forgotten what all of the function arguments mean.  The general formal for a docstring is follows:
```
def name_of_function(argument_1, argument_2):
    """ Less than 80 character description of function

    Input:
        argument_1: (type) brief explanation of argument_1 with enough detail to help
            a new user understand it.  Each line should be fewer than 80 characters.  And 
            subsequent lines should be indented.
        argument_2: (type) same situation as above.

    Returns: 
        Longer explanation of what the function returns.  Should include the type of the 
        object and relevant parameters (i.e. shape, dimension, etc).  Each line of this 
        section should also be fewer than 80 characters. 
    """

Hope that helps! 
```